settings.gradle is missing for gradle build

## How to reproduce

- [ ] run `./gradlew clean azureFunctionsRun`

## logs output

```
$ ./gradlew clean azureFunctionsRun

FAILURE: Build failed with an exception.

* What went wrong:
Project directory 'C:\dev\peaks\opensource\test\src\main\java\azure-functions-samples-java' is not part of the build defined by settings file 'C:\dev\peaks\opensource\test\settings.gradle'. If this is an unrelated build, it must have its own settings file.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 1s
```

--- a/settings.gradle
+++ b/settings.gradle
@@ -1,0 +1,1 @@
+rootProject.name = "azure-functions-samples-java"

